src/Select.js

```diff
@@ -70,7 +71,7 @@ export default class Select extends React.Component {
    */
   componentDidUpdate(prevProps, prevState) {
     if (this.props.options.length > 5 && this.refs.selected && prevState.active === false && this.state.active === true) {
-      this.refs.selected.scrollIntoView();
+      scrollIntoView(ReactDOM.findDOMNode(this.refs.selected));
```

Comment:
Do we need ReactDOM here?

Reply:
Yes, in this case "scrollIntoView" is a method of the SelectOption Component class. So, in short, when I separated scrollIntoView into a lib folder it became a static function that requires a DOM node.